build.xml

```diff
@@ -74,7 +74,7 @@
     <property name="dir.lib.saxon" value="${dir.lib}/saxon"/>
     <property name="dir.lib.xerces" value="${dir.lib}/xerces"/>
     <property name="saxon.download.version" value="SaxonHE11-5"/>
```

Comment:
```suggestion
    <property name="saxon.download.version" value="SaxonHE12-4"/>
```

Reply:
Should be adjusted in the [BUILD_COMMANDLINE.md](https://github.com/music-encoding/music-encoding/blob/develop/BUILD_COMMANDLINE.md) as well.

Reply:
And in the Docker file as well: https://github.com/music-encoding/docker-mei/blob/main/Dockerfile#L12C5-L12C19